ghpages.mk

```diff
@@ -36,19 +36,23 @@ PUSH_GHPAGES ?= true
 endif
 PUSH_GHPAGES ?= false
 
+# Allow overriding the pages branch. It is gh-pages on GitHub, but pages on
+# Forgejo (currently not auto-detected).
+PAGESBRANCH ?= gh-pages
```

Comment:
Renamed (in an insta-squashed fixup).

As for detection: Yes it can, now that GITHUB_HOST is available (I missed that this has now become actionable), added on top. One fewer hack in the patched ghpages.yaml :-)